_includes/tutorial-content.html

```diff
@@ -5,36 +5,31 @@
         <div class="column is-full">
           <h2 class="title tutorial-section-title" id="run-it">Run it</h2>
 
-          {% for step in site.data.harnesses[page.static_data][page.stack].dev.steps %}
+          {% for step in
+          site.data.harnesses[page.static_data][page.stack].dev.steps %}
 
           <div class="tutorial-try-it-step content">
-            <h4 class="subtitle" id="{{ step.title | slugify }}">
+            <h2
+              class="title tutorial-section-item-title"
+              id="{{ step.title | slugify }}"
+            >
+              {{ step.title }}
+            </h2>
+
+            <div class="content-item">
               <div class="num">{{ forloop.index }}</div>
-              <div class="text">{{ step.title }}</div>
-            </h4>
-
-            {% for section in step.content %}
-
-            {% if section.action == "docker_ksql_cli_session" %}
-
-              {% if section.render.skip != true %}
-                {% capture prose %}{% include {{ section.render.file }} %}{% endcapture %}
-                {{ prose | asciidocify }}
-              {% endif %}
-
-              {% for subsection in section.stdin %}
-                {% capture prose %}{% include {{ subsection.render.file }} %}{% endcapture %}
-                {{ prose | asciidocify }}
-              {% endfor %}
-
-            {% elsif section.render.skip != true %}
-
-              {% capture prose %}{% include {{ section.render.file }} %}{% endcapture %}
-              {{ prose | asciidocify }}
-
-            {% endif %}
-
-            {% endfor %}
+              <div class="item-display">
+                {% for section in step.content %} {% if section.action ==
```

Comment:
nit: for consideration add formatting to code blocks here and elsewhere

Reply:
@bbejeck my apologies, I'm not sure what you mean by formatting.  Can you please clarify?

Reply:
I think since it now reformatted into one-liner, it becomes super hard to decipher this.

Reply:
cc: @scott-confluent is working on this in another PR